website/src/pages/index.module.css

```diff
@@ -176,10 +176,6 @@ html[data-theme='dark'] .heroBanner h2 {
     }
 }
 
-.tagline span {
-    color: var(--ifm-color-primary);
-}
```

Comment:
The new `rspack` bundler (or adjacent tooling) seems to wrongly resolve `!important` CSS rules (the tagline had blue words instead of transparent). Removing this rule seems to fix the issue (and not break anything else.)